IdentityCore/src/network/MSIDURLSessionManager.m

```diff
@@ -33,6 +34,8 @@ + (void)initialize
     if (self == [MSIDURLSessionManager self])
     {
         __auto_type configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
+        configuration.timeoutIntervalForResource = MSID_DEFAULT_TIMEOUT_FOR_RESOURCE;
```

Comment:
Why don't we set those values from configuration and remove MSID_DEFAULT_TIMEOUT_FOR_RESOURCE?

Reply:
removing this, as there is no need when the default value we are using is the value defaultSessionConfiguration uses.